pkg/converter/v2/converter.go

```diff
@@ -285,15 +285,14 @@ func (c *converter) updateComponents() error {
 	return nil
 }
 
-func removeNameFromParams(arg *map[string]interface{}) {
-	parameters, ok := (*arg)["parameters"].(map[string]interface{})
+func removeNameFromParams(arg map[string]interface{}) {
+	parameters, ok := arg["parameters"].(map[string]interface{})
 	if !ok {
 		return
 	}
 	for _, rawParam := range parameters {
 		if param, ok := rawParam.(map[string]interface{}); ok {
 			delete(param, "name")
-			rawParam = param
```

Comment:
As maps are already pointers, no need to reassign the var.

I wrote a simple demo code in case anyone wants to check by themselves: https://play.golang.org/